src/components/Home.js

```diff
@@ -17,18 +32,24 @@ class Home extends Component {
     return (
       <div className="Home">
         <h1>Code Blog</h1>
-        <ul>
+
+          <div onClick={this.handleClick}>
```

Comment:
Why is `onCLick` is attached to the `div`. on Click should belong to `Button` belof